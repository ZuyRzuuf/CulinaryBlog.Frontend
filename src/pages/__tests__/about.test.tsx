import { render } from '@testing-library/react';
import AboutPage from '../about';

describe('AboutPage', () => {
    it('matches snapshot', () => {
        const { asFragment } = render(<AboutPage />);
        expect(asFragment()).toMatchSnapshot();
    });
});
