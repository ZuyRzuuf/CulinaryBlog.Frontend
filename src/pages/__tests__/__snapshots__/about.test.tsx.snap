// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutPage matches snapshot 1`] = `
<DocumentFragment>
  <header
    class="bg-gray-800 text-white p-4"
  >
    <nav>
      <a
        href="/"
      >
        <span
          class="mx-2 cursor-pointer"
        >
          Home
        </span>
      </a>
      <a
        href="/recipes"
      >
        <span
          class="mx-2 cursor-pointer"
        >
          Recipes
        </span>
      </a>
      <a
        href="/articles"
      >
        <span
          class="mx-2 cursor-pointer"
        >
          Articles
        </span>
      </a>
      <a
        href="/about"
      >
        <span
          class="mx-2 cursor-pointer"
        >
          About
        </span>
      </a>
      <a
        href="/contact"
      >
        <span
          class="mx-2 cursor-pointer"
        >
          Contact
        </span>
      </a>
    </nav>
  </header>
  <main
    class="p-4"
  >
    <section
      class="bg-cover bg-center h-96 relative text-white"
      style="background-image: url(path/to/about-image.jpg);"
    >
      <div
        class="absolute inset-0 bg-black opacity-50"
      />
      <div
        class="container mx-auto px-4 h-full flex items-center justify-center"
      >
        <div>
          <h1
            class="text-4xl font-bold"
          >
            About PanPrzyGarach
          </h1>
          <p
            class="text-xl mt-2"
          >
            Learn more about our culinary journey and mission
          </p>
        </div>
      </div>
    </section>
    <section
      class="my-16"
    >
      <h2
        class="text-2xl font-bold mb-8"
      >
        Our Mission
      </h2>
      <p
        class="text-lg leading-relaxed"
      >
        At PanPrzyGarach, our mission is to inspire and empower people to discover and create delicious recipes, explore new cuisines, and celebrate their love for food. We strive to provide our visitors with high-quality content, including innovative recipes, informative articles, and expert tips on cooking and baking.
      </p>
    </section>
    <section
      class="my-16"
    >
      <h2
        class="text-2xl font-bold mb-8"
      >
        Meet the Author
      </h2>
      <div
        class="flex items-center"
      >
        <img
          alt="Author Name"
          class="w-32 h-32 rounded-full mr-8"
          src="path/to/author-image.jpg"
        />
        <div>
          <h3
            class="text-xl font-bold"
          >
            Author Name
          </h3>
          <p
            class="text-lg leading-relaxed mt-4"
          >
            Author Name is a professional chef and passionate home cook with over 10 years of experience in the culinary world. With a love for creating innovative recipes and exploring new flavors, Author Name started PanPrzyGarach to share their culinary knowledge and inspire others to embark on their own culinary adventures.
          </p>
        </div>
      </div>
    </section>
  </main>
  <footer
    class="bg-gray-800 text-white p-4"
  >
    <p>
      Â© 2023 PanPrzyGarach
    </p>
  </footer>
</DocumentFragment>
`;
